server/schemas/resolverRoutes/queries.js

```diff
@@ -1,13 +1,43 @@
 const { AuthenticationError } = require('apollo-server-express');
 const { User, Brand, Skater, SkateVideo } = require('../../models');
+const { eventNames } = require('../../models/Soundtrack');
 
 const Query = {
-        me: async (parent, args, context) => {
-          if (context.user) {
-            return User.findOne({ _id: context.user._id });
-          }
-          throw new AuthenticationError('You need to be logged in!');
-        },
+  me: async (parent, args, context) => {
+    if (context.user) {
+      return User.findOne({ _id: context.user._id });
+      }
+      throw new AuthenticationError('You need to be logged in!');
+  },
+  brand: async ( parent , { _id }) => {
+    return await Brand.findById(_id).populate('skateVideos')
+  },
+  brands: async () => {
+    return await Brand.find().populate('skateVideos');
+  },
+  skaters: async () => {
+    return await Skater.find();
+  },
+  skater: async (parent, { _id }) => {
+    return await Skater.findById(_id);
+  },
+  skateVideos: async (parent, { skater, name }) => {
```

Comment:
Why do you need to the skater and name for querying skate videos? The brands and skaters have a reference to the videos. Don't you just need to populate(Skater) populate(SkateVideo)

Reply:
I just did that so you can query skate videos for specific skaters, or the name of the video, its similar to how its set up in the  Redux Code review app, I can update the typedefs to change it so we do not need to require the skaters or name.